### DISCUSSION TITLE
Bunge convention

### DISCUSSION THREAD

**Discussion by RammyCa** [mtex-toolbox/mtex#1887] [Ask Anything]
Hi, I am not sure weather this line on Rotations-Definitions is correct, where you mention:
Euler Angles
"One of the most common ways to describe a rotation is as three subsequent rotations about fixed axes, e.g., first around the z axis, second around the x axis and third again around the z. The corresponding rotational angles are commonly called Euler angles. Beside the most common ZXZ covention other choices of the axes are sometimes used..."

I have some confusion with the first line, where you mention both subsequent rotations and fixed axes. Are you saying you do necesarily subsequent rotations that change the axes for the next rotations or are you saying each rotation happens independently of one another? The term "Fixed" here is my main cause of trouble.

Thank you in advance for your response.

**Comment by JSLecomte** [comment]
Hello Rammyca,
indeed the rotations are successive and the frame changes.
Look at the drawing below, I hope it is clear to you
![Bunge_Convention](https://github.com/mtex-toolbox/mtex/assets/89849334/bb3b824c-be0e-4aa0-967a-63e9e0b6b04a)
